### ISSUE TITLE
npm install raise error

### ISSUE THREAD

**Issue by saicoco** [geeeeeeeeek/electronic-wechat#448]
#### Description

First of all, thanks for your attention to open an issue for this project.
Please notice that if you are requesting a **feature**, then you should give a **brief description** of your request.
If you are reporting a **bug**, please **follow the template** below.
A bug report **without detailed information** required will have a **very low priority** and even be **ignored** (closed directly)!

#### Specifications

- Version of Electron (run `$ electron --version`): `v0.0.0`
- OS: `Ubuntu15.10`
- Stack trace from the error message (if any)

```
npm ERR! git clone git@github.com:github:parro-it/electron-localshortcut fatal: remote error: 
npm ERR! git clone git@github.com:github:parro-it/electron-localshortcut    is not a valid repository name
npm ERR! git clone git@github.com:github:parro-it/electron-localshortcut   Email support@github.com for help
npm ERR! notarget No compatible version found: electron-localshortcut@'github:parro-it/electron-localshortcut#b1119d46c0582263a7c20c1bcd710476dafcfe01'
npm ERR! notarget Valid install targets:
npm ERR! notarget ["0.0.1","0.2.0","0.2.1","0.2.2","0.3.0","0.3.1","0.3.2","0.4.0","0.5.0","0.6.0","0.6.1","1.0.0"]
npm ERR! notarget 
npm ERR! notarget This is most likely not a problem with npm itself.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

npm ERR! System Linux 4.2.0-16-generic
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install"
npm ERR! cwd /home/xxx/APP/electronic-wechat
npm ERR! node -v v0.10.25
npm ERR! npm -v 1.4.21
npm ERR! code ETARGET
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /home/xxx/APP/electronic-wechat/npm-debug.log
npm ERR! not ok code 0
```

**Comment by saicoco** [comment]
The main idea of error is about ` Permission denied (publickey)' when I clone `git@github.com:github:parro-it/electron-localshortcut`, I know that I don't have the privaliage to the repo, however, it must be done. So what should I do~~

**Comment by Kulbear** [comment]
Received.
Closed due to dup #431
